SampleTest/SampleTest.cs

```diff
@@ -305,14 +328,26 @@ void LinearRegression ()
 		#endregion
 		public static void Main (string [] args)
 		{
-			Console.WriteLine (Environment.CurrentDirectory);
+            //need TF# on Windows with GPU support?
+            //find _pywrap_tensorflow_internal.pyd in the tensorflow_gpu-1.1.0-cp36-cp36m-win_amd64.whl
+            //the location in the whl is here: \tensorflow_gpu-1.1.0.data\purelib\tensorflow\python\
+            //extract, copy and rename _pywrap_tensorflow_internal.pyd to libtensorflow.dll
+            //copy to you TF# folder
+            //if running with libtensorflow.dll where Python 3.5 environment is needed - append the py35 environment to the path            
+            //see https://github.com/larcai/cognibooks/blob/master/tensorflow/1_tf_setup.workbook 
+            var envpaths = new List<string> { @"C:\ProgramData\Anaconda3\envs\py35" }
+                .Union(Environment.GetEnvironmentVariable("PATH").Split(new char[] { ';' }, StringSplitOptions.RemoveEmptyEntries));
+            Environment.SetEnvironmentVariable("PATH", string.Join(";", envpaths));
+            
```

Comment:
Why is this needed?   When using the NuGet package, the support library is already included.

Also, I do not want to introduce something with hardcoded paths like that.